api/ipfsproxy/config.go

```diff
@@ -317,7 +317,15 @@ func (cfg *Config) toJSONConfig() (jcfg *jsonConfig, err error) {
 	return
 }
 
-func (cfg *Config) String() string {
-	jcfg, _ := cfg.toJSONConfig()
-	return config.String(*jcfg, nil)
+func (cfg *Config) String() (string, error) {
```

Comment:
Either we keep the Stringer interface by not returning errors (maybe returning the error instead of the string) or we rename this to a different method (ToDisplayJSON).